tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -717,3 +717,160 @@ def _find_output(predictions):
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'scores_out': StringField(description="Scores output layer name."),
+            'boxes_out': StringField(description="Boxes output layer name."),
+        })
+
+        return parameters
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+
+        # Set default values
+        self.min_sizes = [[32, 64, 128], [256], [512]]
+        self.steps = [32, 64, 128]
+        self.variance = [0.1, 0.2]
+        self.confidence_threshold = 0.05
+        self.nms_threshold = 0.3
+        self.keep_top_k = 750
+
+    @staticmethod
+    def calculate_anchors(list_x, list_y, min_size, image_size, step):
+        anchors = []
+        s_kx = min_size / image_size[1]
+        s_ky = min_size / image_size[0]
+        dense_cx = [x * step / image_size[1] for x in list_x]
+        dense_cy = [y * step / image_size[0] for y in list_y]
+        for cy, cx in itertools.product(dense_cy, dense_cx):
+            anchors.append([cx, cy, s_kx, s_ky])
+        return anchors
+
+    def calculate_anchors_zero_level(self, f_x, f_y, min_sizes, image_size, step):
+        anchors = []
+        for min_size in min_sizes:
+            if min_size == 32:
+                list_x = [f_x + 0, f_x + 0.25, f_x + 0.5, f_x + 0.75]
+                list_y = [f_y + 0, f_y + 0.25, f_y + 0.5, f_y + 0.75]
+            elif min_size == 64:
+                list_x = [f_x + 0, f_x + 0.5]
+                list_y = [f_y + 0, f_y + 0.5]
+            else:
+                list_x = [f_x + 0.5]
+                list_y = [f_y + 0.5]
+            anchors.extend(self.calculate_anchors(list_x, list_y, min_size, image_size, step))
+        return anchors
+
+    def prior_boxes(self, feature_maps, image_size):
+        anchors = []
+        for k, f in enumerate(feature_maps):
+            for i, j in itertools.product(range(f[0]), range(f[1])):
+                if k == 0:
+                    anchors.extend(self.calculate_anchors_zero_level(j, i, self.min_sizes[k],
+                                                                     image_size, self.steps[k]))
+                else:
+                    anchors.extend(self.calculate_anchors([j + 0.5], [i + 0.5], self.min_sizes[k][0],
+                                                          image_size, self.steps[k]))
+        anchors = np.clip(anchors, 0, 1)
+
+        return anchors
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        """
+        Args:
+            identifiers: list of input data identifiers
+            raw: output of model
+        Returns:
+            list of DetectionPrediction objects
+        """
+
+        raw_outputs = self._extract_predictions(raw, frame_meta)
+
+        batch_scores = raw_outputs[self.scores_out]
+        batch_boxes = raw_outputs[self.boxes_out]
```

Comment:
Do you have specific reason for it?
SSDONNXAdapter have regex only due to MO limitations on model conversion. As I see the original output nodes saved in the model and there is no way to replace them using regex. (Please explain how do you will replace 342 and 353 with regex?)
I do not prefer mask out the errors if outputs were changed. After us, the customers can get the same error

Reply:
It is not necessary for this model, but could be useful in future. Furthermore, it expands functionality through low cost improvements.

Nevertheless, it is just my advice, make your own decision.

Reply:
We add it if it will be necessary in the future, now this is no need in these changes, so better to stay as is.
Majority of converters works without regex and more over it can be also painful for quantization if we will use them (during statistics collection we have more outputs than in usual cases, user can not know all used names and there are among them several which can be matched to regex or not. It will be bad surprise for users that they can not quantize model). It is why I think explicitly setting of output names better than regular expressions. Any way for support new model we need to write config for it, so these changes will be added to config.